Automattic/wp-calypso
# Remove the ultimate traffeic marketing guide from Calypso

_Note: do not merge this until approved by the marketing team._

#### Changes proposed in this Pull Request
- Make marketing section isomorphic
- Move redirects from the client side to the server side
- Add redirect for the ultimate traffic marketing guide to point at the new course website
- Remove the ultimate traffic marketing guide from Calypso

#### Testing instructions

1. Load the calypso.live URL
2. `/marketing/ultimate-traffic-guide` should redirect to https://wpcourses.com/course/intro-to-search-engine-optimization-seo/'
3. `marketing/do-it-for-me` should redirect to wordpress.com/do-it-for-me
5. The rest of the /marketing section should work as expected.
6. The /sharing section should also work as expected. (Compare against production to see if there are differences.)
